Formula/befunge93.rb

```diff
@@ -0,0 +1,18 @@
+class Befunge93 < Formula
+  desc "Esoteric programming language"
+  homepage "http://catseye.tc/projects/befunge93/"
```

Comment:
On my network, the above url automatically linked to the URL you mentioned. I'll change it anyways.